src/ansys/geometry/core/designer/body.py

```diff
@@ -1324,12 +1324,14 @@ def plot(
         **plotting_options: Optional[dict],
     ) -> None:  # noqa: D102
         # lazy import here to improve initial module load time
+        from ansys.tools.visualization_interface.types.mesh_object_plot import MeshObjectPlot
 
-        from ansys.geometry.core.plotting import PlotterHelper
+        from ansys.geometry.core.plotting import GeomPlotter
 
-        PlotterHelper(use_trame=use_trame).plot(
-            self, merge_bodies=merge, screenshot=screenshot, **plotting_options
-        )
+        meshobject = MeshObjectPlot(self, self.tessellate(merge=merge))
+        pl = GeomPlotter(use_trame=use_trame)
+        pl.plot(meshobject, **plotting_options)
+        pl.show(screenshot=screenshot)
 
```

Comment:
Hmm... plotting options are only necessary to be passed to plot?

Reply:
`show` method should have a kwargs argument with any extra arguments the backend might have. Now that I see it, this is confusing in this case, since we only have one **kwargs for two functions really. Maybe I should add a `show_options` optional argument to specify the extra options for `show`